### DISCUSSION TITLE
Does this EFI work for Big Sur (dell e7450)?

### DISCUSSION THREAD

**Discussion by devwithshahzaib** [SkyrilHD/Dell-Exx50-Hackintosh#65] [Q&A]
is this EFI supported for installing Big Sur on my Dell e7450?

**Comment by SkyrilHD** [comment]
Yes, Big Sur is supported and should work without any major modifications (with Monterey EFI). However, if you want to use the Intel WiFi, you need to replace the Monterey AirportItlwm kext with the Big Sur one. You can find the releases of AirportItlwm [here](https://github.com/OpenIntelWireless/itlwm/releases)

**Reply by devwithshahzaib** [reply]
Ok, after replacing this it'll be fully working? and I'm good to go to install bigSur?

**Reply by SkyrilHD** [reply]
Yeah

**Reply by devwithshahzaib** [reply]
I've followed following steps.
1- Downloaded the Hacintosh BigSur from olarila.com.
2- Using balenaEtcher Flashed my Usb drive.
3- Using miniPartition tool and Explorer++ copied the EFI file to my drive EFI Partition.
4- I'll replace the Monterey AirportItlwm kext with the Big Sur one.

so far i'm doing fine?

**Reply by SkyrilHD** [reply]
I recommend following this guide: https://dortania.github.io/OpenCore-Install-Guide/installer-guide/windows-install.html#making-the-installer

Any other method of getting the installer _can_ work, however I don't recommend them.

Otherwise, the steps are fine

**Reply by devwithshahzaib** [reply]
Thank's for your time😊! I'll try and inform you what happens.

**Reply by devwithshahzaib** [reply]
Hey mate, I'm facing restart in loop showing apple logo and continuously restarting. 

I've successfully erased my drive and selected it and installed after installing it keeps restarting...🙁

**Reply by devwithshahzaib** [reply]
Fortunately I've successfully installed BigSur but my wifi, keypad click is not working. Any solution please 🥺

**Reply by SkyrilHD** [reply]
For WiFi: https://github.com/SkyrilHD/Dell-Exx50-Hackintosh#intel-wifi and https://github.com/SkyrilHD/Dell-Exx50-Hackintosh/discussions/65#discussioncomment-5456775

For Trackpad: disable Force Touch in System Preferences

**Reply by devwithshahzaib** [reply]
There's written no trackpad found!

I used ApplePS2SmartTouchpad.kext and enabled it in config.plist but it's very slow and click is not working plus same error trackpad not found...
